component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java

```diff
@@ -395,6 +396,33 @@ private void checkTotpEnabled(AuthenticationContext context, String username) th
         }
     }
 
+    /**
+     * Update the secret key in the context with value in database before validate.
+     *
+     * @param context  Authenticated context.
+     * @param username Authenticated users' username
```

Comment:
Missing a `.` at the end of the statement.

Reply:
Address with - https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/113/commits/e27180b645c234705cdc784dd90e0eaef49943fc